code/modules/keybindings/setup.dm

```diff
@@ -52,13 +57,16 @@
 			var/key = macro_set[k]
 			var/command = macro_set[key]
 			winset(src, "[setname]-[REF(key)]", "parent=[setname];name=[key];command=[command]")
-		winset(src, "[setname]-say", "parent=[setname];name=T;command=[say]")
-		winset(src, "[setname]-me", "parent=[setname];name=M;command=[me]")
-		winset(src, "[setname]-ooc", "parent=[setname];name=O;command=[ooc]")
-		if(use_tgui_say)
-			winset(src, "[setname]-radio", "parent=[setname];name=Y;command=[radio]")
-			winset(src, "[setname]-looc", "parent=[setname];name=L;command=[looc]") // NSV13 - Moves LOOC keybind to L from U
-			winset(src, "[setname]-close-tgui-say", "parent=[setname];name=Escape;command=[tgui_say_create_close_command()]")
+		// If we bind these, we're going to disable input relay.
+		// This *does* silently break TGUI-Say, but I doubt players who want to use this mode care. -Francinum
+		if(!using_chat_relay)
```

Comment:
```suggestion
		if(using_chat_relay)
```
This does the opposite right now (the `preftoggle hotkeys` var is used for people who are in *hotkey mode* and is enabled by default).
You may want to rename the variable (or invert it when checked initially) but I will push a change to edit the ! out myself now to fix this on live.